pages/takeHomeInternDayToDay.md

```diff
@@ -18,14 +18,14 @@ Improving the Take Home Application is a lot easier to say then to actually do.
-It is paramount that while you are discovering how the Take Home application works you keep the mindset of documenting your precise steps you have takin and pit falls you have avoided so that interns that join the team after you can quickly become familiar with the applications code without making the same mistakes you did. If you don’t document, more than likely new mobile interns are going to make the same mistakes. If you are improving the Take Home code, writing a new script or doing anything to improve the quality of life of the Take Home project and you come across a step or guide that you think others would benefit from, please document it. 
+It is paramount that while you are discovering how the Take Home application works you keep the mindset of documenting your precise steps you have taken and pit falls you have avoided so that interns that join the team after you can quickly become familiar with the applications code without making the same mistakes you did. If you don’t document, more than likely new mobile interns are going to make the same mistakes. If you are improving the Take Home code, writing a new script or doing anything to improve the quality of life of the Take Home project and you come across a step or guide that you think others would benefit from, please document it. 
 
 I don’t know how to stress the importance of this point any further. The 3 months, 18 hours a week, is very short, so any steps to help other interns to make their time more efficient will have a ripple affect long after you are done with this internship.
 
 > “You will get out of this internship only what you put into it” – [Waryhermit](profiles/waryhermit.md) (Former Intern Lead)
 
 You can document your findings by either adding to [existing guides](takehome.md) (including this guide) or by creating new ones.
-I know some of yous English aint so good, which is completely fine. You can still be an effective intern by improving the Take Home code and solving issues for the Take Home application. But, If you come across an important improvement to a guide or want to create a new guide, but don’t feel confident in your ability to convey it in a guide you can then ask some who is confident to listen to it in voice chat, [talk.ole.org](talk.ole.org), and they can then convert it into a guide/new step for you. Just don’t wear out your welcome with this option on other interns.
+I know some of your English ain't so good, which is completely fine. You can still be an effective intern by improving the Take Home code and solving issues for the Take Home application. But, If you come across an important improvement to a guide or want to create a new guide, but don’t feel confident in your ability to convey it in a guide you can then ask some who is confident to listen to it in voice chat, [talk.ole.org](talk.ole.org), and they can then convert it into a guide/new step for you. Just don’t wear out your welcome with this option on other interns.
```

Comment:
Haha! @kylemathias Yeah, I see the effect the author was going for, but it can easily get lost on the reader if their english isn't great to begin with. What a meta-problem.